### PR TITLE
[gobgp] initial integration

### PR DESCRIPTION
**Pull request by 0x34d**
[GoBGP](https://osrg.github.io/gobgp/) is a major Golang implementation of BGP.

Used by: `CloudWan`, `Tele2`, `Sakura`, `NexusGuard`, `FastNetMon`, `DigitalOcean`.

@fujita Does this look good?

### CODE REVIEW

File: projects/gobgp/project.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+homepage: "https://osrg.github.io/gobgp"
+language: go
+primary_contact: ""
```
Comment (0x34d) [inline projects/gobgp/project.yaml:3]: @fujita, may I have your email ID for the `oss-fuzz` bug reporting system?

**Comment by jonathanmetzman** [comment]
We accept this project if upstream does.

**Comment by jonathanmetzman** [comment]
We accept this project if upstream does.

**Comment by 0x34d** [comment]
> We accept this project if upstream does.

@jonathanmetzman Done https://github.com/osrg/gobgp/issues/2721

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**